Testscripts/Linux/utils.sh

```diff
@@ -3559,4 +3559,35 @@ function Run_SSHCommand()
 			ssh "${ip}" "${cmd}"
 		fi
 	done
+}
+
+function Check_lsvmbus()
+{
+	# check if lsvmbus exists, or the running kernel does not match installed version of linux-tools
+	lsvmbus_path=$(which lsvmbus)
```

Comment:
I think 'whereis' command is better here. In SLES 15 SP1,  I get the path of lsvmbus from 'whereis' command, but 'which' does not.
lisa@LISAv2-OneVM-juhlee2862-MF42-20200805001700-role-0> lsvmbus_path=$(which lsvmbus)
which: no lsvmbus in (/home/lisa/bin:/usr/local/bin:/usr/bin:/bin:/usr/lib/mit/bin)
lisa@LISAv2-OneVM-juhlee2862-MF42-20200805001700-role-0> echo $lsvmbus_path

lisa@LISAv2-OneVM-juhlee2862-MF42-20200805001700-role-0> whereis lsvmbus
lsvmbus: /usr/sbin/lsvmbus

I think you are actually checking the file path.

Reply:
Yeah, you are right. We couldn't get the path from "which lsvmbus" under "lisa" user, and lsvmbus can't run under "lisa". 
lisa@LISAv2-OneVM-lbh-LR16-20200805205414-role-0:~> lsvmbus
Absolute path to 'lsvmbus' is '/usr/sbin/lsvmbus', so running it may require superuser privileges (eg. root).

If it's root user, we can get the path.
LISAv2-OneVM-lbh-LR16-20200805205414-role-0:/home/lisa # which lsvmbus
/usr/sbin/lsvmbus
LISAv2-OneVM-lbh-LR16-20200805205414-role-0:/home/lisa # lsvmbus
VMBUS ID  1: [Dynamic Memory]
VMBUS ID  2: Synthetic IDE Controller
VMBUS ID  3: Synthetic IDE Controller
VMBUS ID  4: Synthetic mouse
VMBUS ID  5: Synthetic keyboard

This function is moved from LSVMBUS.sh and verify_vmbus_heartbeat_properties.sh, I put them into a function in utils.sh